README.md

```diff
@@ -1,26 +1,37 @@
 ![Logo](https://raw.githubusercontent.com/zendframework/zf2/234b554f2ca202095aea32e4fa557553f8849664/resources/ZendFramework-logo.png)
 
-# Welcome to the *Zend Framework 2.6* Release!
+# Welcome to the *Zend Framework 3.0* Release!
 
 ## RELEASE INFORMATION
 
-*Zend Framework 2.6.0dev*
+*Zend Framework 3.0.0dev*
 
-This is the sixth minor (feature) release for the version 2 series.
+This is the third major release for Zend Framework.
 
 DD MMM YYYY
 
-### UPDATES IN 2.6.0
+### UPDATES IN 3.0.0
 
 Please see [CHANGELOG.md](CHANGELOG.md).
 
 ### SYSTEM REQUIREMENTS
 
-Zend Framework 2 requires PHP 5.5 or later; we recommend using the
+Zend Framework 3 requires PHP 5.6 or later; we recommend using the
 latest PHP version whenever possible.
 
 ### INSTALLATION
 
+We no longer recommend installing this package directly. The package is a
```

Comment:
I'd like to see the recommended installation process noted here - presumably we recommend starting with the skeleton?